code/game/objects/items/weapons/knuckledusters.dm

```diff
@@ -0,0 +1,84 @@
+/obj/item/melee/knuckleduster
+	name = "knuckleduster"
+	desc = "Simple metal punch enhancers, perfect for bar brawls."
+	icon = 'icons/obj/knuckleduster.dmi'
+	icon_state = "knuckleduster"
+	flags = CONDUCT
+	force = 5
+	throwforce = 3
+	w_class = WEIGHT_CLASS_SMALL
+	resistance_flags = FIRE_PROOF
+	materials = list(MAT_METAL = 500)
+	origin_tech = "combat=2"
+	attack_verb = list("struck", "bludgeoned", "bashed", "smashed")
+	var/gripped = FALSE
+	var/elite = FALSE
+	var/robust = 10
+	var/trauma = 5
+
+/obj/item/melee/knuckleduster/attack_self(mob/user)
+	if(!gripped)
+		gripped = TRUE
+		to_chat(user, "You tighten your grip on [src], ensuring you won't drop it.")
+		flags |= NODROP
+		return
+	if(gripped)
+		gripped = FALSE
+		to_chat(user, "You relax your grip on [src].")
+		flags &= ~NODROP
+		return
+
+/obj/item/melee/knuckleduster/attack(mob/living/target, mob/living/user, obj/item/organ/external, obj/item/organ/internal/)
+	. = ..()
+	if(!ishuman(target))
+		return
+
+	var/obj/item/organ/external/punched = target.get_organ(user.zone_selected)
+	if(gripped && prob(robust) && target.health < 90) // Better at throwing strong punches when gripped
+		if(isslimeperson(target))
```

Comment:
It would be better if you did check using TRAIT_NO_BONES, since your objective here is to affect those without bones and not simply slime people.